### PR TITLE
PILOT-1151: Add project-files routes

### PR DESCRIPTION
**Pull request by vmoshynskyi**
## Summary

Add project-files routes.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1151

## Type of Change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Testing

Are there any new or updated tests to validate the changes?

- [x] Yes
- [ ] No

### CODE REVIEW

**Review by mloshakov — CHANGES_REQUESTED** [review]
All looks great, just a few question

File: app/api_registry.py [hunk 1]
```diff
@@ -66,4 +82,4 @@ def api_registry(app: FastAPI):
     app.include_router(api_workbench.router, prefix="/v1")
     app.include_router(resource_request.router, prefix="/v1")
     app.include_router(file_stats.router, prefix="/v1")
-    app.include_router(file_search.router, prefix="/v1")
+    app.include_router(api_project_files.router, prefix='/v1')
```
[the remark this answers is not in the document]
Comment (vmoshynskyi) [reply app/api_registry.py:85]: Sure, I'll change them to `"`.

File: services/search/client.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    async def _get(self, url: str, params: Mapping[str, Any]) -> Response:
+        logger.info(f'Calling search service {url} with query params: {params}')
+
+        try:
+            response = await self.client.get(url, params=params)
+            assert response.is_success
+        except Exception:
+            message = f'Unable to query data from search service with url "{url}" and params "{params}".'
+            logger.exception(message)
+            raise SearchServiceException(message)
```
[the remark this answers is not in the document]
Comment (vmoshynskyi) [reply services/search/client.py:51]: Usually each client should have it's own scope of exceptions with a base class. I'd say that it's a blueprint for the future development of the search client.

File: config.py [hunk 3]
```diff
@@ -140,7 +140,6 @@ def modify_values(self, settings):
         settings.DOWNLOAD_SERVICE_CORE_V2 = settings.DOWNLOAD_SERVICE_CORE + '/v2/'
         settings.DOWNLOAD_SERVICE_GR_V2 = settings.DOWNLOAD_SERVICE_GR + '/v2/'
         settings.KG_SERVICE = settings.KG_SERVICE + '/v1/'
-        settings.SEARCH_SERVICE = settings.SEARCH_SERVICE + '/v1/'
```
[the remark this answers is not in the document]
Comment (vmoshynskyi) [reply config.py:143]: I've moved this into client scope.
https://github.com/PilotDataPlatform/bff-web/blob/c0205483481170aa3825aefc4fd1a5c32eac13bf/services/search/client.py#L39
It's a client responsibility to know which route should be used. All other code just works with the client methods.

**Review by mloshakov — APPROVED** [review]

**Pull request merged**